docs/milestones.md

```diff
@@ -0,0 +1,21 @@
+# Milestones
+
+## Milestone 0: Análisis del problema y primer acercamiento
+
+Analizar las historias de usuario. Decidir las tecnologías que serán necesarias para comenzar a tratar el problema. Con respecto a la historia de usuario 1, plantear la manera de calcular los turnos de los enfermeros.
+
++ **Mínimo viable**
```

Comment:
Te enlazo otro de los problemas frecuentes recogidos en el archivo correspondiente:

_Por favor, no inventéis y pidáis cosas que no vienen a cuento ni sirven realmente para nada. Estoy pensando en pedir "documentos" con el modelo del problema, pero no descarto que metáis diagramas UML o alguna otra cosa igualmente inútil. Un producto es código funcionando, y que siga algún criterio de viabilidad. Punto._

En resumen: el PMV debe ser código.